insights/specs/datasources/ls.py

```diff
@@ -52,7 +54,13 @@ def list_with_la_filtered(broker):
 
 @datasource(HostContext)
 def list_with_lan(broker):
-    return _list_items(Specs.ls_lan_dirs)
+    filters = _list_items(Specs.ls_lan_dirs)
+    if 'specs.fstab_mounted' in filters and FSTab in broker:
+        filter_list = filters.split()
+        for mntp in broker[FSTab].mounted_on.keys():
+            mnt_point = os.path.dirname(mntp)
+            filter_list.extend([mnt_point] if mnt_point and mnt_point not in filter_list else [])
```

Comment:
declear the `filter_list` as a set first to avoid duplicate element by nature. 
```python
filter_list = set(_list_items(Specs.ls_lan_dirs)) # assume the _list_items returns `list` here already
for mntp in broker[FSTab].mounted_on.keys():
    filter_list.add(os.path.dirname(mntp))


```